src/ophys_etl/modules/maximum_projection/utils.py

```diff
@@ -0,0 +1,294 @@
+import pathlib
+import h5py
+import numpy as np
+import multiprocessing
+import multiprocessing.managers
+import scipy.ndimage as scipy_ndimage
+
+
+def scale_to_uint8(img: np.ndarray) -> np.ndarray:
```

Comment:
Is this function not somewhere else in the codebase?

Reply:
I don't think so. We definitely implemented it as a part of `staging/segmentation_dev`. I think it's an open question whether and how much of that branch ever gets merged into `main`.

I'll put this somewhere more generally accessible.

Reply:
Actually: never mind.

`utils/array_utils/normalize_array`

Reply:
There was apparently also already a `downsample_array` method. I've changed the code to use that, rather than my custom-implemented `decimate_video` method.